app/src/main/java/io/apicurio/registry/mt/TenantManagerClientProducer.java

```diff
@@ -75,12 +75,13 @@
                             "but the no auth properties aren't properly configured");
                 }
 
+                ApicurioHttpClient httpClient = new JdkHttpClientProvider().create(properties.getTenantManagerAuthUrl().get(), Collections.emptyMap(), null, new AuthErrorHandler());
```

Comment:
Why not use `ApicurioHttpClientFactory` here?

Reply:
That was my plan here, but that fails in native mode, the implementations of the SPI are not loaded. I have asked the Quarkus team to see if I can find the reason, but no luck yet. That's why I'm using directly the JDK provider. Sorry, probably worth a comment in the code explaining it.

Reply:
ah OK that makes sense.